Simplenote/ar.lproj/Localizable.strings

```diff
@@ -87,7 +87,7 @@
 "Delete Tag" = "حذف وسم";
 
 /* Signup Support Text */
-"Didn't get an email? You may already have an account. Contact %1$@ for help." = "ألم يصلك بريدٌ إلكترونيٌ؟ ربما يكون لديك حساب بالفعل. اتصل بـ %1$@ للحصول على مساعدة.";
+"Didn't get an email? There may already be an account associated with this email address. Contact %1$@ for help." = "Didn't get an email? There may already be an account associated with this email address. Contact %1$@ for help.";
```

Comment:
The new beta lane runs the action to update the localizations. That's why this and all the other files changed.

There's no actual translated value because this string is still untranslated in [GlotPress](https://translate.wordpress.com/projects/simplenote/macos/).